### ISSUE TITLE
sort out paypal donations button

### ISSUE THREAD

**Issue by CIEspost** [zacck-zz/Menanawe#36]
@zacck christiaan asked me to add make my paypal able to accept donations. please can you help me with that, I want to just do that and send out that video in the next sprint just so that I have that thing sent out. I am not gonna spend mcuh time on MenaNawe. I decided i jsut want to send out that 360 video and as soon as the donations button is sorted, because i know its gonna be super long till i can send these things out properly and i just need to send out the 360 video for now so that it doesnt get too old. Your throughts?

The email fro Christiaan:

"Hey Carla

Het jy jou PayPal opgestel om donations te kan aanvaar? Die donate buttons moet link na 'n PayPal page. Dit word opgestel in jou PayPal account, soos ek dit verstaan, maar ek het geen ervaring daarmee nie.

Ek gaan die week weer van server verander so ek gaan weer jou domain IP adres moet verander. Jy kan vir my jou Cloudflare details stuur of jy kan dit self verander na 139.162.214.152

Sal die week aandag gee om die website klaar te maak.

Groete"

**Comment by zacck-zz** [comment]
@CIEspost  yes I can sort this out let's plan it and put it on a sprint

**Comment by CIEspost** [comment]
@zacck this is the only menanawe thing i want to do for this month, because i need to finish this thing and christian is willing to finish it this week so please can we do this this week, it should be quick i assume?

**Comment by zacck-zz** [comment]
@CIEspost  yes I think he just needs your paypal account we can most likely fix this, this week

**Comment by zacck-zz** [comment]
@CIEspost  do you have a paypal account ?

**Comment by zacck-zz** [comment]
If you have one please open this link https://www.paypal.com/us/buttons/paymentbuttons  and sign in then from there I can help you

**Comment by CIEspost** [comment]
@zacck i logged in, what now?

**Comment by CIEspost** [comment]
now we wait for feedback from christiaan
